Jint/Runtime/Interpreter/Statements/JintDoWhileStatement.cs

```diff
@@ -50,6 +50,11 @@ protected override Completion ExecuteInternal(EvaluationContext context)
                     }
                 }
 
+                if (context.Engine._isDebugMode)
```

Comment:
I think we could promote the value to `context.DebugMode` and initialize in context constructor to make code cleaner?

Reply:
Checked, and `context.DebugMode` is already there - it's just only used once. I've changed all the references to _isDebugMode outside `Engine`. Only one that's a small issue is:

https://github.com/sebastienros/jint/blob/0e315f41df03339c51928bdc49b9786b940cbf03/Jint/Native/Function/ScriptFunctionInstance.cs#L144

I've changed that one to `_engine._activeExecutionContext.DebugMode` for consistency - now, only `Engine` and the context constructor read the `_isDebugMode` field. Could declare `context` as a local variable, since _activeExecutionContext is also used a few lines earlier - however, I'm not actually sure if the `OrdinaryCallEvaluateBody()` call might conceivably change the value of it - `DebugMode` won't care, but anything else that might decide to use that local `context` variable would.

Looking at the others (all agreed) tonight.